conda/common/logic.py

```diff
@@ -781,13 +781,39 @@ def minimize(self, objective, bestsol=None, trymax=False):
 
         return self._minimize(objective_list, bestsol=bestsol, trymax=trymax)
 
+    # TODO: test code, remove this!
+    _problem_write_out_condition = None
+    # 1. obtain problematic instance via:
```

Comment:
Don't try to let `conda` solve that command all the way: You will most likely run out of memory!

I used this script during for local testing:
```python
if __name__ == '__main__':
    from conda.gateways.logging import initialize_logging, set_conda_log_level, TRACE
    initialize_logging()
    set_conda_log_level(TRACE)

    from json import load

    from conda.common.logic import Clauses, CryptoMiniSatSolver
    Clauses._problem_write_out_condition = None

    with open('problem.json') as f:
        problem = load(f)

    m = max(max(map(abs, cl)) for cl in problem['clauses'])
    c = Clauses(m=m, sat_solver=CryptoMiniSatSolver)
    c.add_clauses(problem['clauses'])

    newsol = c.minimize(problem['objective'], problem['bestsol'])
```
to re-run a single `minimize` call that has been written out by this code.